examples/wolfram/plugin.asterai.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+import "node_modules/@asterai/sdk/protobuf/asterai.proto";
+
+service Wolfram {
+  //provides simple textual answers for any valid WolframAlpha query in natural language. is able to also provide the market price for commodity metals such as gold,
+  //silver and palladium. this method can only provide the answer for one question or one price at a time.
+  rpc textQuery(TextQueryRequest) returns (TextQueryResponse) {}
+}
+
+message TextQueryRequest {
+  //the url-encoded query to send to WolframAlpha
```

Comment:
Please add a spacing here and begin the comment with a capital letter, just for consistency.

Reply:
Addressed